.github/workflows/tests.yml

```
@@ -64,43 +64,43 @@ jobs:
 
             if (label_names.includes('CI-syntax-only')) {
               console.log('CI-syntax-only label found. Skipping tests.')
-              console.log("::set-output name=run-tests::false");
+              console.log('::set-output name=run-tests::false');
             }
             else {
               console.log('No CI-syntax-only label found. Running tests.')
-              console.log("::set-output name=run-tests::true");
+              console.log('::set-output name=run-tests::true');
             }
 
             if (label_names.includes('CI-force-arm')) {
               console.log('CI-force-arm label found. Running ARM builds.')
-              console.log("::set-output name=skip-arm-tests::false");
+              console.log('::set-output name=skip-arm-tests::false');
```

```suggestion
              console.log('::set-output name=force-arm-tests::true');
```

Is there a reason not to be consistent with the labels and outputs? I find all the negatives confusing. Plus, if the label isn't present we're not always skipping tests, we're just not forcing them.

---

I agree that it's confusing and simply setting variables like `syntax-only` and `force-arm` that follow the label names (like the other examples) would make this easier to follow. My latest commit changes the names and modifies the values and `tests` conditions accordingly.